include/mqt-core/dd/Package.hpp

```diff
@@ -1786,7 +1845,7 @@ template <class Config> class Package {
     auto r = trace(a, eliminate);
     return cn.lookup(r);
   }
-  ComplexValue trace(const mEdge& a) {
+  template <class Edge> ComplexValue trace(const Edge& a) {
```

Comment:
I would slightly prefer to make this a `template <class Node>` and use `Edge<Node>` in order to get better IDE and code completion support because then the compiler at least knows that the parameter is some kind of Edge.

Reply:
I changed the code as suggested.